### PR TITLE
Push metrics from Tenant Fetcher

### PR DESCRIPTION
**Pull request by pkosiec**
**Description**

Changes proposed in this pull request:

- Push metrics from Tenant Fetcher

**Related issue(s)**
#1235

### CODE REVIEW

File: components/director/internal/tenantfetcher/service.go [hunk 1]
```diff
@@ -11,8 +11,8 @@ import (
 	"github.com/pkg/errors"
 )
 
-//go:generate mockery -name=TenantStorageService -output=automock -outpkg=automock -case=underscore
 type TenantStorageService interface {
+	//go:generate mockery -name=TenantStorageService -output=automock -outpkg=automock -case=underscore
```
Comment (kfurgol) [inline components/director/internal/tenantfetcher/service.go:15]: Was that change intentional?

File: components/director/internal/metrics/pusher.go [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+	return &Pusher{
+		eventingRequestTotal: eventingRequestTotal,
+		pusher:               pusher,
+		instanceID:           instanceID,
+	}
+}
+
+func (p *Pusher) RecordEventingRequest(method string, statusCode int, desc string) {
+	log.WithField(InstanceIDKeyName, p.instanceID).Infof("Recording request with status code '%d'...", statusCode)
```
Comment (kfurgol) [inline components/director/internal/metrics/pusher.go:43]: Maybe log also the description? Now the logs doesn't say a lot:

```
time="2020-05-21T10:32:36Z" level=info msg="Recording request with status code '0'..." func="github.com/sirupsen/logrus.(*Entry).Logf" file="/go/pkg/mod/github.com/sirupsen/logrus@v1.4.2/entry.go:314" instance=e8d108b8-1841-42aa-9bfe-f1bf55cee28e
```

File: components/director/cmd/tenantfetcher/README.md [hunk 3]
```diff
@@ -33,3 +33,4 @@ The Tenant Fetcher binary allows you to override some configuration parameters.
 | **APP_MAPPING_FIELD_DISCRIMINATOR** |               | Optional name of the field in the event data payload used to filter created tenants. If provided, only events containing this field with a value specified in **APP_MAPPING_VALUE_DISCRIMINATOR** will be used. |
 | **APP_MAPPING_VALUE_DISCRIMINATOR** |               | Optional value of the discriminator field for filtering created tenants. It is used only if **APP_MAPPING_FIELD_DISCRIMINATOR** is provided.                                                                    |
 | **APP_TENANT_PROVIDER**             |               | Tenant provider name                                                                                                                                                                                            |
+| **APP_METRICS_PUSH_ENDPOINT**      |               | Optional Prometheus Pushgateway endpoint for pushing application metrics                                                                                                                                                 |
```
Comment (klaudiagrz) [inline components/director/cmd/tenantfetcher/README.md:36]:
```suggestion
| **APP_METRICS_PUSH_ENDPOINT**      |               | Optional Prometheus Pushgateway endpoint for pushing Application metrics                                                                                                                                                 |
```
